grape-swagger-rails.gemspec

```diff
@@ -15,7 +15,11 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
   spec.files         = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   spec.require_paths = %w[lib]
-  spec.add_dependency 'railties', '>= 6.0.6.1'
+
+  rails_version = ENV.fetch('RAILS_VERSION', '>= 6.0.6.1')
+  rails_version = '>= 0' if rails_version == 'edge'
```

Comment:
There's no way to specify a github dependency in a gemspec. Not 100% sure whether `>= 0` is the way to go here but it appears to do the trick.

Reply:
Don't modify the gemspec to be dependent on ENV, that is a moving target.

Instead here say `spec.add_dependency 'railties'`, and then in Gemfile pick whichever version you want, which can be based on ENV.

Reply:
Good point. I started out with completely locking the version in here, then proceeded to override it in the Gemfile later. I think I can just simply revert the changes to the gemspec.